src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -72,11 +72,11 @@ angular.module("ovh-angular-mondial-relay")
         /**
          * Reformat the opening hours of the relays
          * @param {Object} opening Structure describing opening times {monday: [{start: "0800", end:"1900"}]}
-         * @returns {Array} List of open days [{day: "monday", opening: ["08:00-19:00"]}]
+         * @returns {Array} List of open days [{days: ["monday", "tuesday"], hours: ["08:00-19:00"]}]
          */
         var reformatOpening = function reformatOpening (opening) {
             var reformatTime = function (openingTime) {
-                return openingTime.substring(0, 2) + ":" + openingTime.substring(2, 4);
+                return openingTime.substring(0, 2) + "h" + (openingTime.substring(2, 4) === "00" ? "" : openingTime.substring(2, 4));
```

Comment:
"h" is a french notation
":" is an ISO notation https://en.wikipedia.org/wiki/ISO_8601 

Why not use the vanilla Date formater? https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Date/toLocaleTimeString

Reply:
Btw, as I format the time regarding zeros (12h00 becomes 12h), it has to be relevant to keep 'h'. But as you say, it's a french notation, so...dunno